### PR TITLE
legal evidence statements and general cleanup

### PR DESCRIPTION
**Pull request by chadsmall**

### CODE REVIEW

File: source/sections/11-incident_response_policy.md [hunk 1]
```diff
@@ -57,27 +57,28 @@ Bind employees must report any unauthorized or suspicious activity seen on produ
...
+       6. The lead member of the SIRT team facilitates initiation of a [SIR Form](sections/sir-form.doc) or an [Incident Survey Form](sections/IH-Survey.pdf). The intent of the SIR form is to provide a summary of all events, efforts, and conclusions of each Phase of this policy and procedures. TODO: Consider using restricted Notion template, page listing for these instead of pdf. Or consider into a template field in the Incident Jira issue type itself.  
 3. The Security Officer, Privacy Officer, or Bind representative appointed notifies any affected Customers and Partners. If no Customers and Partners are affected, notification is at the discretion of the Security and Privacy Officer.
 4. In the case of a threat identified, the Security Officer is to form a team to investigate and involve necessary resources, both internal to Bind and potentially external.
+5. Be diligent collecting evidence and retaining it to support potential legal action in the event that there is a follow-up action after the incident.
```
Comment (chadsmall) [inline source/sections/11-incident_response_policy.md]: This was the driver statement for enhancing our policy to be more clear for our support given a direct question from Medtronic assessment

File: source/sections/11-incident_response_policy.md [hunk 2]
```diff
@@ -27,7 +27,7 @@ The incident response process addresses:
 
 ## 11.2 Incident Management Policies
 
-The Bind incident response process follows the process recommended by [SANS](https://www.sans.org), an industry leader in security. Process flows are a direct representation of the SANS process which can be found in [this document](sections/incident-flowchart.pdf).
+The Bind incident response process follows the process recommended by [SANS](https://www.sans.org), an industry leader in security. Process flows are a direct representation of the SANS process which can be found in [this document](incident-flowchart.pdf).
```
Comment (gleneiden) [inline source/sections/11-incident_response_policy.md]: The incident-flowchart.pdf looks to still be in the sections directory.  Did you intend to remove "sections/" from the document?

File: source/sections/11-incident_response_policy.md [hunk 3]
```diff
@@ -57,27 +57,28 @@ Bind employees must report any unauthorized or suspicious activity seen on produ
    1. Direct report to management, the Security Officer, Privacy Officer, or other;
    2. Email;
    3. Phone call;
-   4. Online incident response form located [here](https://docs.google.com/a/catalyze.io/forms/d/1Hn4di9Jdw5JT8vISMh6tVUnh94VpeHucell7Ca4fKTo/viewform); TODO: way to report incident online (jira?)
+   4. Online incident response issue located in Jira [here](https://thebind.atlassian.net/servicedesk/customer/portal/1/create/29);
    5. Secure Chat.
    6. Anonymously through workforce members desired channels.
```
Comment (gleneiden) [inline source/sections/11-incident_response_policy.md]: 1.  For "Secure Chat" are we telling people to use Slack direct messaging?  I'm not suggesting we name slack in the policy, just wondering if that is the solution we'll use to support #5, or if we will need something different?
2.  Do we have a way to support anonymous reporting?  If not, can we remove that option, or do we need to support an anonymous path for increased likelihood of getting reports?

File: source/sections/11-incident_response_policy.md (hunk 1 above)
Comment (gleneiden) [inline source/sections/11-incident_response_policy.md]: Do we want to be more specific about "be diligent?"  Is the intent to completely **capture** the evidence that is collected, or is the intent to ensure the discovery and collection of evidence is **thorough**?

**Comment by chadsmall** [comment]
Good points Glen - I was a little quick to merge - I'll open a new PR and get these things in.

File: source/sections/11-incident_response_policy.md (hunk 2 above)
Comment (chadsmall) [inline source/sections/11-incident_response_policy.md]: These links will have to be all updated depending on where our policies reside long-term.  These were built for a ruby automated deployment.  We'll probably want to update to Notion links in the short-term and maybe something else once we automate.

File: source/sections/11-incident_response_policy.md (hunk 3 above)
Comment (chadsmall) [inline source/sections/11-incident_response_policy.md]: 1.  Yes, was thinking slack.  
2.  I was thinking anonymous could be a note dropped into some private HR cabinet that has come up with Aly.  We can remove if we don't think that is spelled out enough at this point.

**Pull request merged**